samples/Sandbox/Sandbox.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
```

Comment:
The sandbox thus becomes platform dependent

Reply:
Are you sure? I think this only means hiding the console window in .netcore 3+?

Reply:
Oops, yeah. I misinterpreted ([1](https://github.com/dotnet/sdk/blob/afe72f4eca6065e945705293cb6b33a31fa1dc50/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Common.targets#L36), [2](https://github.com/dotnet/sdk/blob/afe72f4eca6065e945705293cb6b33a31fa1dc50/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets#L457)). 'WinExe' should be interpreted as something like 'non-console EXE'...

Reply:
That property controls the `Subsystem` field of `IMAGE_OPTIONAL_HEADER` of PE executable. .NET images always come in PE executable format.